docs/rules.md

```diff
@@ -363,6 +364,18 @@ These are considered invalid:
     end
 
 
+### NoInterpolationOnlyStrings
+
+A rule that disallows strings which are entirely the result of an
+interpolation.
+
+Good:
+  output = inspect(__MODULE__)
+
+Bad:
+  output = "Dogma.Rule.NoInterpolationOnlyStrings"
```

Comment:
TODO: how to write something like `"#{inspect var}"` in the `@moduledoc` without having `var` be interpreted